examples/kitchen-sink/public/_redirects

```diff
@@ -1,2 +1,3 @@
 /custom-widget    /index.html   200
 /modal    /index.html   200
+/side-panel    /index.html   200
```

Comment:
This is unrelated to this change so consider it non-blocking, but don't we probably want some catch-all here `/*  /index.html   200` ?